### PR TITLE
First pass HTML and CSS

### PR DESCRIPTION
**Pull request by reimerharts**
- Added the Roboto font.
- Added some section deviders.
- Changed some H tags for more SEO and easier customization.
- Added BG image.
- Got started on the CSS with the focus of mobile first.

### CODE REVIEW

File: src/main/resources/templates/charsheet.html [hunk 1]
```diff
@@ -39,57 +41,53 @@
...
       </div>
 
+      <div class="section_devider"></div>
+
       <div class="section" id="mid">
         <div class="stat" id="level">
-          <span id="levelLabel" class="statLabel">LV</span>
-          <span id="levelValue" class="statValue" th:text="${character.level}" th:if="${!admin}">1</span>
+          <span id="levelLabel" class="statLabel"><p>LV</p></span>
+          <span id="levelValue" class="statValue" th:text="${character.level}" th:if="${!admin}"><p>1</p></span>
```
Comment (xathien) [inline src/main/resources/templates/charsheet.html]: These `th:text` fields (used for Thymeleaf templating) will break with your new nested elements, as will all of the JS that updates these fields as you spend skill points or as the admin adjusts levels, etc., etc.

If you move the `th:text` and the `id` attributes into the nested `p` element for each of them, they Should™ continue working.

File: src/main/resources/templates/charsheet.html [hunk 2]
```diff
@@ -39,57 +41,53 @@
...
-            <span class="statLabel" th:id="|${stat.literal}Label|" th:text="${#strings.toUpperCase(stat.literal)}"></span>
-            <span class="statValue" th:id="|${stat.literal}Value|" th:text="${character.stats.get(stat).get()}">10</span>
+            <span class="statLabel" th:id="|${stat.literal}Label|" th:text="${#strings.toUpperCase(stat.literal)}"><p></p></span>
+            <span class="statValue" th:id="|${stat.literal}Value|" th:text="${character.stats.get(stat).get()}"><p>10</p></span>
           </div>
         </div>
-
       </div>
 
+      <div class="section_devider"></div>
```
Comment (xathien) [inline src/main/resources/templates/charsheet.html]:
```suggestion
      <div class="section_divider"></div>
```

File: src/main/resources/static/css/charsheet.css [hunk 3]
```diff
@@ -1,10 +1,46 @@
...
+  padding: 16px;
+  color: white;
+  background: url(../images/bg3.png) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+#main .section_devider {
```
Comment (xathien) [inline src/main/resources/static/css/charsheet.css]:
```suggestion
#main .section_divider {
```

**Review by xathien — CHANGES_REQUESTED** [review]
I'm also not seeing anywhere that you're using the new font files; am I missing something?

File: src/main/resources/templates/charsheet.html (hunk 1 above)
Comment (reimerharts) [reply src/main/resources/templates/charsheet.html]: Alright good to know!

**Comment by reimerharts** [comment]
> I'm also not seeing anywhere that you're using the new font files; am I missing something?

Not yet. I didn't get to that in the first pass.

File: src/main/resources/static/css/charsheet.css [hunk 4]
```diff
@@ -13,65 +49,78 @@ body {
...
 
-span.skillValue, span.specRanks {
+span .skillValue, span .specRanks {
 	display: inline-block;
 	width: 30px;
 	text-align: right;
 }
 
-input.specName {
+input .specName {
```
Comment (xathien) [inline src/main/resources/static/css/charsheet.css]: Before, the CSS rule was "input with class `specName`". Now you have it as "input with a child with class `specName`", which won't ever evaluate.

(This comment applies to the 4 changes above this one, too)

Comment (reimerharts) [reply src/main/resources/static/css/charsheet.css]: Got it. I don't remember why I changed that... 
Changed it back.

File: src/main/resources/templates/charsheet.html [hunk 5]
```diff
@@ -15,21 +15,23 @@
 			admin = [[${admin}]];
 			urlPrefix = [[${urlPrefix}]];
     </script>
-    <link rel="stylesheet" href="/css/charsheet.css"
-          th:href="@{/css/charsheet.css}" media="screen" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/charsheet.css"
+          th:href="@{charsheet.css}" media="screen" />
```
Comment (xathien) [inline src/main/resources/templates/charsheet.html:20]: Let's make absolutely sure we put this back... 🙂 
```suggestion
          th:href="@{/css/charsheet.css}" media="screen" />
```

**Pull request merged**